Source/Details/ASBasicImageDownloader.h

```diff
@@ -25,14 +25,15 @@ NS_ASSUME_NONNULL_BEGIN
 @interface ASBasicImageDownloader : NSObject <ASImageDownloaderProtocol>
 
 /**
- * A shared image downloader which can be used by @c ASNetworkImageNodes and @c ASMultiplexImageNodes
+ * A shared image downloader which can be used by @c ASNetworkImageNodes and @c ASMultiplexImageNodes.
+ * The userInfo provided by this downloader is `nil`.
  *
  * This is a very basic image downloader. It does not support caching, progressive downloading and likely
  * isn't something you should use in production. If you'd like something production ready, see @c ASPINRemoteImageDownloader
  *
  * @note It is strongly recommended you include PINRemoteImage and use @c ASPINRemoteImageDownloader instead.
  */
-+ (instancetype)sharedImageDownloader;
++ (ASBasicImageDownloader *)sharedImageDownloader;
```

Comment:
We could also change it to:

@property (class, readonly, strong) ASBasicImageDownloader * sharedImageDownloader;

Would improve code completion.

Reply:
Great call, doing it